test/test_translator.py

```diff
@@ -138,6 +117,36 @@ def test_parse_polyline() :
         ]
     assert  test_coordinates == valid_coordinates
 
+def test_parse_polyline_invalid_data() :
+    test_polyline= '' 
+    test_coordinates=icone_translator.parse_polyline(test_polyline)
+    valid_coordinates= []
+    assert  test_coordinates == valid_coordinates
+
+    test_polyline= None
+    test_coordinates=icone_translator.parse_polyline(test_polyline)
+    valid_coordinates= []
+    assert  test_coordinates == valid_coordinates
+
+    test_polyline= 'invalid' 
+    test_coordinates=icone_translator.parse_polyline(test_polyline)
+    valid_coordinates= []
+    assert  test_coordinates == valid_coordinates
+
+    try:
+        test_polyline= 'a,b,c,d' 
+        test_coordinates=icone_translator.parse_polyline(test_polyline)
+        assert False
+    except RuntimeError: 
+        assert True
+
+
+
+
+
+
+    
+
 
 def test_get_road_direction():
```

Comment:
Same as above, these should to be broken out to test a single unit of logic with appropriate names to the methods so we know what is being tested.

Reply:
added additional edge cases with valid test function name and pushed into dev.